# Define goal
hd_servo/MotorSequence motor_sequence
---
# Define the result
bool success
---
# Define a feedback message
float32 percentage_complete
